app/screens/receive-bitcoin-screen/receive-btc.tsx

```diff
@@ -278,10 +276,30 @@ const ReceiveBtc = () => {
     }
   }, [paymentRequest, LL])
 
+  useEffect(() => {
+    if (state === PaymentRequestState.Paid) {
+      ReactNativeHapticFeedback.trigger("notificationSuccess", {
+        ignoreAndroidSystemSettings: true,
+      })
+    } else if (state === PaymentRequestState.Error) {
+      ReactNativeHapticFeedback.trigger("notificationError", {
+        ignoreAndroidSystemSettings: true,
+      })
+    }
+  }, [state])
+
   if (!paymentRequestDetails || !setAmount) {
     return <></>
   }
 
+  const togglePaymentRequestType = () => {
+    const newPaymentRequestType =
+      paymentRequestDetails.paymentRequestType === PaymentRequest.Lightning
+        ? PaymentRequest.OnChain
+        : PaymentRequest.Lightning
+    setPaymentRequestType(newPaymentRequestType, true)
```

Comment:
I know this is just a code moving around, but using object as input to the function would make it more clear what `true` is used for here. 

the only way to know currently is to look at the function. with an object, the `true` would have more context also here.

Reply:
fixed it